.circleci/fetch_geth_parity_solc.sh

```diff
@@ -45,7 +45,7 @@ if [[ ! -x ${PARITY_PATH} ]]; then
       fi
   fi
 fi
-ln -sf ${PARITY_PATH} ${LOCAL_BASE}/bin/parity
+ln -sfn ${PARITY_PATH} ${LOCAL_BASE}/bin/parity
```

Comment:
How doesn `-n` make the symlinking more robust?

```
       -n, --no-dereference
              treat LINK_NAME as a normal file if it is a symbolic link to a directory
```

Reply:
If for some reason the target already exists and is a symlink to a directory `ln` will create the new link inside that directory. Using `-n` forces it to not dereference the symlink and instead overwrite the symlink.

Reply:
Aha I see. That makes sense. Thank you man.